### ISSUE TITLE
multiplexing subscriptions not working after updating nchan

### ISSUE THREAD

**Issue by qhenkart** [slact/nchan#448]
I just updated nchan from version 1.0.8 to 1.1.14 and my channel multiplexing set-up has stopped working. I've fiddled with it for hours and looked through the change logs, but I can't find any reason that it stopped receiving messages. If I turn off the second channel then it works again, and if I downgrade back to 1.0.8 everything works as expected.

Here is the multiplexing set up
```
    nchan_subscriber;
    nchan_subscriber_first_message newest;
    nchan_subscriber_timeout 0;
    nchan_channel_id $sent_http_x_channel_id profile_$sent_http_x_profile_id;
    nchan_use_redis on;
```

```
  nchan_publisher;
  nchan_channel_id $1;
```

with a basic publisher

I should add I see no difference from the entries in 1.0.8 to 1.1.14 when I look at the redisdb

I am publishing to only the first channel_id

**Comment by qhenkart** [comment]
I am publishing to only the first channel_id

**Comment by qhenkart** [comment]
I've narrowed it down to being the presence of both multiplexing and the nchan_subscriber_first_message. I will report back further findings

**Comment by slact** [comment]
I'll look into this, thanks for the report.

**Comment by slact** [comment]
This should now be fixed in 1.1.15, or in master

**Comment by slact** [comment]
Great! Thanks for verifying it. The new version will be out in at most 2 weeks.
